include/seqan3/std/span.hpp

```diff
@@ -0,0 +1,421 @@
+// -*- C++ -*-
+//===------------------------------ span ---------------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is dual licensed under the MIT and the University of Illinois Open
+// Source Licenses. See LICENSE.TXT for details.
+//
+//===---------------------------------------------------------------------===//
+
+
+/*!\file
+// \brief Provides std::foobar from the C++20 standard library.
+// \see https://en.cppreference.com/...
+*/
+
+//!\cond
+// #include <__config>
```

Comment:
whats this?

Reply:
Not sure, was in the original file and I didn't want to remove it, but removing it now!